.github/workflows/reusable_image_release.yaml

```diff
@@ -0,0 +1,251 @@
+name: Reusable Pre Release and Release Pipeline for docker images
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        description: 'Full repository name (org/repo)'
+        type: string
+        required: true
+      docker_platforms:
+        description: 'Comma-separated list of Docker platforms'
+        type: string
+        default: 'linux/amd64,linux/arm64'
+      docker_image_name:
+        description: 'Docker image name to use'
+        type: string
+        required: true
+      integration_name:
+        type: string
+        required: false
+        default: ''
+      s_three_path:
```

Comment:
took me a second to understand what path it is referring to :D

Reply:
+1 I do not like it that much